docker-compose.yml

```diff
@@ -3,7 +3,7 @@ services:
     image: "postgres:latest"
     restart: always
     volumes:
-      - $HOME/MobSF/postgresql_data:/var/lib/postgresql
+      - $HOME/MobSF/postgresql_data:/var/lib/postgresql/data
```

Comment:
Why explicitly mount `data`?

Reply:
I've just tested it and found that without such modification it doesn't work. As I remember mounted folder was empty after db creation.
From official documentation(https://hub.docker.com/_/postgres):
![image](https://github.com/user-attachments/assets/45404878-a10c-4e1d-a9f9-9fd86ebde428)

Similar issue: https://stackoverflow.com/a/41650891

Reply:
Sounds good.